src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -903,9 +914,38 @@ private static bool HasImplicitNullableConversion(TypeSymbol source, TypeSymbol
                 return true;
             }
 
+            if (HasImplicitTupleConversion(unwrappedSource, unwrappedDestination, ref useSiteDiagnostics))
+            {
+                return true;
+            }
+
             return false;
         }
 
+        private bool HasImplicitTupleConversion(TypeSymbol source, TypeSymbol destination, ref HashSet<DiagnosticInfo> useSiteDiagnostics)
+        {
+            ImmutableArray<TypeSymbol> sourceTypes;
+            ImmutableArray<TypeSymbol> destTypes;
+
+            if (!source.TryGetElementTypesIfTupleOrCompatible(out sourceTypes) ||
```

Comment:
Please add a test for public API Compilation.ClassifyConversion when it is called with underlying types.